src/ui/map.js

```diff
@@ -436,6 +436,13 @@ class Map extends Camera {
         if (typeof window !== 'undefined') {
             window.addEventListener('online', this._onWindowOnline, false);
             window.addEventListener('resize', this._onWindowResize, false);
+            if (typeof window.screen.orientation === 'object') {
+                // ensures browser supports latest ScreenOrientation API
+                window.screen.orientation.addEventListener('change', this._onWindowResize, false);
+            } else {
+                // If not then use deprecated 'orientationchange' event
+                window.addEventListener('orientationchange', this._onWindowResize, false);
```

Comment:
As far as I can see, `window` `orientationchange` event [is a part of a living standard](https://developer.mozilla.org/en-US/docs/Web/API/Window/orientationchange_event), it's not deprecated. So you can get rid of the branching and only use this one. Perhaps it was mistaken for [screen orientationchange](https://developer.mozilla.org/en-US/docs/Web/API/Screen/onorientationchange), which is indeed deprecated?

Reply:
Yep mistake on my part, have included this change in my last commit.